components/lib/picklist/PickListSubList.js

```diff
@@ -78,6 +84,7 @@ export const PickListSubList = React.memo(
                             onMouseDown={(event) => props.onOptionMouseDown({ ...event, index, type: props.type })}
                             ptm={ptm}
                             cx={cx}
+                            onMouseMove={() => changeFocusedItemOnHover(index)}
```

Comment:
This is a total nitpick but I like when the `on` events grouped together so move this 2 lines up under `onMouseDown`  If you don't want to I understand

Reply:
Done